client/src/components/ChangesModal.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react'
+import { 
+    Modal,
+    Box,
+    Typography,
+    Grid
+ } from '@mui/material'
+ import { IoWarningOutline } from "react-icons/io5";
```

Comment:
I'd prefer if we kept our icon usage in MUI's icon library if possible so we don't have to take on other dependencies.

MUI icon suggestion:
https://mui.com/material-ui/material-icons/?query=warn&theme=Rounded

Reply:
@Spiteless  fixed